.github/workflows/nix.yaml

```
@@ -62,9 +56,8 @@
         id: check-docker-push
         run: |
           # Moving the docker push conditions here from the build job
-          if [[ "${{ github.ref }}" == "refs/heads/main" || 
-                "${{ github.ref }}" == "refs/heads/prodHotPush-Common" || 
-                "${{ github.event.pull_request.merged }}" == "true" ]]; then
+          if [[ "${{ github.ref }}" == "refs/heads/frfs-quote-fix" || 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)